### ISSUE TITLE
Child templates shouldn't ask you to rewrite every block

### ISSUE THREAD

**Issue by Eijebong** [djc/askama#37]
Example:

base.html
```
{% block foo %}{% endblock %}
{% block bar %}Default{% endblock %}
```

foo.html
```
{% extends "base.html" %}
{% block foo %}Foo{% endblock %}
```

Should compile.

Right now it fails because base generate a traits that has a method for each block and the child one doesn't implement the `bar` one.
I had a quick look at the code but can't find any easy way of fixing that :/

**Comment by Eijebong** [comment]
Ok, I thought a bit about it, would you accept a PR that fixes it but parses the base template during the code generation of the child ?

**Comment by djc** [comment]
Fixed in fc7be3c927d2078a317797c0d5477ad608c4c625.
